README.md

```diff
@@ -278,6 +293,29 @@ If the install has been performed as part of snap package, daemon status can be
 
 `systemctl status snap.auto-cpufreq.service.service`
 
+### Update - auto-cpufreq update
+
+Update to the [latest version](https://github.com/AdnanHodzic/auto-cpufreq/releases) of auto-cpufreq with fixes and resolved bugs. 
+
+* If auto-cpufreq is not installed using `snapd`:
+
+    Manually clone auto-cpufreq to the latest release if this mode is not available in your package(follow [Installing auto-cpufreq](#installing-auto-cpufreq)
+
+    Update the package by running(available with both snapd and non-snapd packages):
+
+    `sudo auto-cpufreq --update`
+
+    This does the equivalent of:
+    * `systemctl stop auto-cpufreq && systemctl disable auto-cpufreq`
+    * Fetching new update and install
+    * `systemctl enable auto-cpufreq`
+
+* For snap and AUR packages, run:
```

Comment:
Let's remove lines 313 to 318. Because if they try to do this they'll get a message with auto-cpufreq

Reply:
done